miqa/core/models/frame.py

```diff
@@ -38,3 +39,15 @@ def size(self) -> int:
     @property
     def experiment(self) -> Experiment:
         return self.scan.experiment
+
+    @property
+    def is_in_s3(self) -> bool:
+        return self.raw_path.startswith('s3://')
+
+    @property
+    def s3_download_url(self) -> Optional[str]:
+        if not self.is_in_s3:
+            return None
+        bucket, key = self.raw_path.strip()[5:].split('/', maxsplit=1)
```

Comment:
I generally use [urlparse](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) instead of using string functions. It is more verbose though so ¯\\\_(ツ)\_\/¯